Doc/library/shelve.rst

```diff
@@ -160,18 +181,20 @@ Restrictions
    modules.  The *dict* object passed to the constructor must support those
    methods.  This is generally accomplished by calling one of
    :func:`!bsddb.hashopen`, :func:`!bsddb.btopen` or :func:`!bsddb.rnopen`.  The
```

Comment:
By the way, we also need to update this sentence (from `bsddb` to `berkeleydb`). `bsddb` is deprecated according to https://www.jcea.es/programacion/pybsddb.htm.

Reply:
It looks like the change would be bigger: it’s a new module (although the docs are not very clear) with maybe a new API.

Updating or deprecating this should be discussed in its own ticket 🙂

Reply:
Yes, I agree on it. It would be better to open a new ticket to discuss this issue.

Reply:
... or consider instead opening topic on Discourse; this may need a larger audience than what you'd get on the bug tracker.